### ISSUE TITLE
Standardize image text

### ISSUE THREAD

**Issue by rebeccabilbro** [foxbook/atap#52]
Image titles should be short, titlecase (uppercase), and one line.
The alt text should describe the figures (only). All other interpretive/descriptive text should be in paragraph form.

**Comment by rebeccabilbro** [comment]
This is being done by the copyeditor.
